gc/base/ParallelDispatcher.cpp

```diff
@@ -225,28 +225,32 @@ MM_ParallelDispatcher::kill(MM_EnvironmentBase *env)
 {
 	OMR::GC::Forge *forge = env->getForge();
 
-	if(_workerThreadMutex) {
+	if (_workerThreadMutex) {
```

Comment:
we are very inconsistent with this, but I believe we should have explicit 'NULL != ...'